src/app/app.component.ts

```diff
@@ -93,7 +97,30 @@ export class AppComponent implements OnInit, AfterViewInit {
             "description": "Volumes / Buckets / File Share / Hosts",
             "routerLink": "/block",
             "joyrideStep" : "menuResource",
-            "text" : "View and manage Buckets, Volumes, Volume Groups, File shares and Hosts that have been manually created or applied for through service templates."
+            "text" : "View and manage Buckets, Volumes, Volume Groups, File shares and Hosts that have been manually created or applied for through service templates.",
```

Comment:
do we really need from line100-123 ??
There is no change in admin/tenant resource provisioning

Reply:
The menu items are populated in two different objects for Admin and other tenants. 
This is required.

Reply:
I know, This is populated for two different objects, Admin and other tenants. (Even previously was also doing same.). And moreover router link is same...so basically, it goes to default router link only for admin and tenants. This is input for optimization.

Reply:
Sure. Will look into it at a later date. This will require updating the menu creation and population which is out of scope of the current PR. Could you raise an issue for this. we could track it there.

Reply:
#1277 raised to track this